src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/matcher/RoleNameMatcher.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.matcher;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignment;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseTypeDefinition;
+import uk.gov.hmcts.ccd.domain.model.definition.RoleToAccessProfileDefinition;
+
+import java.util.List;
+
+@Slf4j
+@Component
+public class RoleNameMatcher implements RoleAttributeMatcher {
+
+    @Override
+    public boolean matchAttribute(RoleAssignment roleAssignment, CaseDetails caseDetails) {
+        return true;
```

Comment:
can we define a default method in RoleAttributeMatcher instead of overriding here

Reply:
done